src/components/Gradebook/index.jsx

```diff
@@ -418,6 +418,13 @@ export default class Gradebook extends React.Component {
                 <small className="form-text text-muted search-help-text">Search by username, email, or student key</small>
               </div>
             </div>
+            <div>
+              <span>Selected </span>
```

Comment:
spans here and on lines 424 & 426 seem redundant

If the whole point is to force spaces, you can do that all in 423 & 425's spans and save some markup noise